### PR TITLE
when errors happen check if they were actually conflicts

### PR DESCRIPTION
**Pull request by mafintosh**

### CODE REVIEW

File: test/conflicts.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+  await unreplicate(streams)
+
+  replicate(c, b, t)
+
+  await t.exception(() => b.get(4))
+
+  // await b.update()
+  // await b.get(2)
+  // await a.truncate(0, 1)
```
Comment (kasperisager) [inline test/conflicts.js:35]: Leftover?

File: test/conflicts.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    t.is(length, 5, 'conflict at 5 seen by b')
+  })
+
+  await b.get(2)
+
+  await unreplicate(streams)
+
+  replicate(c, b, t)
+
+  await t.exception(() => b.get(4))
```
Comment (kasperisager) [inline test/conflicts.js:31]:
```suggestion
  await t.exception(b.get(4))
```

Should be sufficient given that the value is a promise that'll reject.

**Review by kasperisager — CHANGES_REQUESTED** [review]

File: test/conflicts.js (hunk 1 above)
Comment (mafintosh) [reply test/conflicts.js:35]: ah yes

Comment (mafintosh) [reply test/conflicts.js]: fixed

File: test/conflicts.js (hunk 2 above)
Comment (mafintosh) [reply test/conflicts.js]: fixed

**Pull request merged**